### PR TITLE
Add subheadings to UG and standardise UG command formats

### PR DESCRIPTION
**Pull request by brein62**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -76,27 +76,31 @@ iVolunteer is your dedicated application for volunteer coordination, designed wi
...
+  e.g. if the command specifies `n/NAME p/PHONE`, `p/PHONE n/NAME` is also acceptable.
 
 * Extraneous parameters for commands that do not take in parameters (such as `vlist`, `elist`, and `exit`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
 * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
 </box>
 
-### Viewing help: `help`
+### Help command
```
Comment (ruo-x) [inline docs/UserGuide.md:87]: ooo isit these subheadings will appear at table of contents?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -573,13 +579,13 @@ coming soon
...
-| **Create a new volunteer profile**            | `vcreate n/VOLUNTEER_NAME p/PHONE_NUMBER e/EMAIL [s/SKILLS]...`<br> e.g.,`vcreate n/John Lim p/81234567 e/john123@gmail.com s/Cooking`                                                                                                                                                                                                  |
+| **Add a quantity of materials to an event**   | `eaddm eid/EVENT_INDEX m/MATERIAL_WITH_QUANTITY`<br> e.g., `eaddm eid/1 m/10 potato`                                                                                                                                                                                                                                                    |
+| **Create a new volunteer profile**            | `vcreate n/NAME p/PHONE e/EMAIL [s/SKILL]...`<br> e.g.,`vcreate vn/John Lim hp/81234567 e/john123@gmail.com s/Cooking`                                                                                                                                                                                                                  |
```
Comment (ruo-x) [inline docs/UserGuide.md:588]: oops theres a `vn/` and `hp` in the example

**Review by ruo-x — CHANGES_REQUESTED** [review]
one small parameter issue at the command summary, but nice work!

File: docs/UserGuide.md (hunk 2 above)
Comment (brein62) [reply docs/UserGuide.md:588]: lol can i resolved a merge conflict then that happened

File: docs/UserGuide.md (hunk 1 above)
Comment (brein62) [reply docs/UserGuide.md:87]: yepp

File: docs/UserGuide.md (hunk 2 above)
Comment (ruo-x) [reply docs/UserGuide.md]: OKI

**Review by tiongMax — APPROVED** [review]
LGTM!

**Review by ruo-x — APPROVED** [review]
HEN HAO HEN HAO CHAO JI WU DI

**Review by jordankanghm — APPROVED** [review]
walao u made me wet...

**Pull request merged**